### ISSUE TITLE
Graphical issues with Soul Calibur

### ISSUE THREAD

**Issue by JacobAL** [p1pkin/demul#364]
On the old renderer, many textures don't show up and trying fixes like disable opaque/translucent and disable autosort just make the menus look worse rather than fix anything.

Not sure with the new renderer because it brings up massive scanlines that make the game very difficult to play. Any way to fix that?

**Comment by p1pkin** [comment]
yeah, it is very simple and obvious: never install new version into folder with some previous version files, never put .ini or other files from older version into new one.

this will fix both mentioned issues (and potentially TONS of other problems)

**Comment by JacobAL** [comment]
This is my first install of demul.

Interestingly though, playing on my laptop has none of the same graphical glitches. Is it because I have an Nvidia card on the PC but AMD on the laptop?

Maybe I'll try reinstalling.

**Comment by p1pkin** [comment]
hard to say for sure, but yes, yours problem looks like graphics card and/or it's drivers specific.

**Comment by JacobAL** [comment]
Ah well, what a shame. It's only SC that's been having issues so far so it's not a massive issue.

**Comment by p1pkin** [comment]
can you tell your GPU model and driver version ?

**Comment by JacobAL** [comment]
EVGA GeForce GTX 970 4GB SSC ACX 2.0+ Video Card
Driver is the most recent but I can get you the full details of it in a little while. I'll get a picture of what's going on with the graphics too.

**Comment by JacobAL** [comment]
Specs: http://i.imgur.com/xgBeIKI.png
Current driver: http://i.imgur.com/cwAqSTe.png
Graphical issues with old renderer: http://i.imgur.com/GMqMkBU.jpg
Graphical issues with new renderer: http://i.imgur.com/PcqsgHL.jpg
Issues with the new renderer are only happening if I stop the emulation and reboot the game for some reason. Interestingly, the new renderer is now allowing me to play SC in VGA mode which may be because I just switched from using CDIs to GDIs.
I think that since the game (if booted once and no more times) seems to work fine now with the new renderer, there is no issue for me any more.

Thanks for the help, sorry to have troubled you.

**Comment by p1pkin** [comment]
> Issues with the new renderer are only happening if I stop the emulation and reboot the game for some reason

that's the problem, there is still some bugs in emulator, so after Stop/Start such artifacts can be seen on NV GPUs. try to avoid this.
thanks for pics and info.
